File: PumpUI/src/app/models/update-pump.dto.ts
export interface UpdatePumpDto {
  Id: number;
  PumpName: string;
  MaxPressure: number | null;
  LiquidTemperatureCelsius: number | null;
  WeightInKilograms: number | null;
  PumpDescription: string;
  ImageUrlPath: string | null;
  PriceInRubles: number;
  MotorForeignKey: number;
  HousingMaterialForeignKey: number;
  WheelMaterialForeignKey: number;
}

File: PumpApi/PumpApi/Models/UpdatePumpDto.cs
namespace PumpApi.Models
{
    public class UpdatePumpDto
    {
        public int Id { get; set; }
        public string PumpName { get; set; }
        public double? MaxPressure { get; set; }
        public double? LiquidTemperatureCelsius { get; set; }
        public double? WeightInKilograms { get; set; }
        public string PumpDescription { get; set; }
        public string ImageUrlPath { get; set; }
        public decimal PriceInRubles { get; set; }
        public int MotorForeignKey { get; set; }
        public int HousingMaterialForeignKey { get; set; }
        public int WheelMaterialForeignKey { get; set; }
    }
}
